tidalapi/session.py

```diff
@@ -460,10 +463,10 @@ def login_session_file(
         if not self.check_login():
             if do_pkce:
                 log.info("Creating new session (PKCE)...")
-                self.login_pkce()
+                self.login_pkce(fn_print=fn_print)
             else:
                 log.info("Creating new session (OAuth)...")
-                self.login_oauth_simple()
+                self.login_oauth_simple(function=fn_print)
```

Comment:
I suggest naming both functions fn_print or similar.

Reply:
Hello, sure, I will do that. I didn't do that before just to avoid to break compatibility in case some user is using named parameters. Anyway for me it's ok to name that parameter consistently.

Reply:
Great! I will make sure to fix this in mopidy-tidal. I'll also add this to the changelog as BREAKING.